src/terms/anyonic.jl

```diff
@@ -0,0 +1,91 @@
+# Ref https://arxiv.org/pdf/1901.10739.pdf
+# This term does not contain the kinetic energy, which must be added separately
+# /!\ They have no 1/2 factor in front of the kinetic energy,
+#     so for consistency the added kinetic energy must have a scaling of 2
+# Energy = <u, ((-i∇+βA)^2 + V) u>
+# where ∇∧A = 2π ρ, ∇⋅A = 0 => A = x⟂/|x|² ∗ ρ
+# H = (-i∇+βA)² + V - 2β x⟂/|x|² ∗ (βAρ + J)
```

Comment:
Space before / after +

Reply:
not going to bother. These are implementation comments recalling the formulas in the (obscure) paper. If it keeps you up all nights by all means go ahead and format it

Reply:
Ok. I'll do it ...